draft-irtf-panrg-path-properties.md

```diff
@@ -124,7 +124,7 @@ Path element:
 Path:
 : A sequence of adjacent path elements over which a packet can be transmitted, starting and ending with a node.
 
-  Paths are unidirectional and time-dependent, i.e., the sequence of path elements over which packets are sent from one node to another may change.
+  Paths are unidirectional and time-dependent, i.e., there can be a variety of paths from one node to another, and the path over which packets are transmitted may change during the lifetime of a flow.
```

Comment:
I like this change since it should cause less confusion than before (i.e., that the sequence of path elements of a path can change).

One small thing is that we now define path using the terminology "flow" while we also define flow using the terminology "path", so there is some form of circular dependency ;D
I don't think this is really a problem but still, we could maybe omit the term flow here:
```suggestion
  Paths are unidirectional and time-dependent, i.e., there can be a variety of paths from one node to another, and the path over which packets are transmitted may change.
```

Reply:
Right, perhaps it's good to not have a "circular dependency" here. And I agree that we can actually omit the last part.